packages/forms-web-app/src/views/selected-appeal/appeal.njk

```diff
@@ -1,43 +1,48 @@
 {% extends "layouts/no-banner-link/main.njk" %}
 
- {% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
- {% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
-	{% from  "pins/components/appeal-sections-block.njk" import appealSectionsBlock %}
-
- {% set title="
- Appeal " + appeal.appealNumber + " - Appeal a planning decision - GOV.UK" %}
-
- {% block pageTitle %}
-   {{ title }}
- {% endblock %}
-
- {% block content %}
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-full">
-
-       {% if errorSummary %}
-         {{ govukErrorSummary({
-           titleText: "There is a problem",
-           errorList: errorSummary
-         }) }}
-       {% endif %}
-
-       <h1 class="govuk-heading-l">Appeal {{appeal.appealNumber}}</h1>
-
-		{{ govukSummaryList({
-			classes: "appeal-headlines govuk-summary-list--no-border",
-			rows: appeal.headlineData
-		}) }}
-
-		<hr class="govuk-section-break govuk-section-break--visible govuk-section-break--m">
-
-
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
```

Comment:
Looks like I did more than I did here. A lot of this was indented by an extra space so I just sorted out the formatting while I was here. The only new bit is inside `{% if appeal.decision %}`

Reply:
Hiding whitespace on GitHub does wonders for seeing actual changes :)